comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -5,10 +5,20 @@
 using System.Text;
 using System.Threading.Tasks;
 
+public class ImportantValues
```

Comment:
`ImportantValues` is not a very descriptive name.

Also not sure why these are in a separate class and not just stored in `DirectoryUtils`?